platform-includes/enriching-events/add-attachment/python.mdx

```diff
@@ -1,28 +1,27 @@
-Attachments live on the `Scope` and will be sent with all events.
+Because attachments live on the [`Scope`](/platforms/python/enriching-events/scopes/), the SDK sends all attachments on a given `Scope` with every non-transaction event captured within the `Scope`. It's also possible to send an attachment with all events, including transactions by setting the `add_to_transactions` option to `True`.
```

Comment:
you very likely want a `<PlatformLink>` here instead of the plain md link

Reply:
It's not necessary for Python because we don't have guides for different platforms, but probably a good idea to do this

Reply:
made platformlink. takes the same time as commenting :-)

Reply:
Thanks @antonpirker 🥔